contracts/javascore/score-util/src/main/java/ibc/icon/score/util/ByteUtil.java

```diff
@@ -33,4 +36,24 @@ public static byte[] join(byte[]... data) {
 
         return result;
     }
+
+    public static byte[] convertBytesToStringBytes(String hexString) {
+        var hexStringInBytes = StringUtil.hexToBytes(hexString);
+        List<Integer> intArr = new ArrayList<>();
+        for (byte b : hexStringInBytes) {
+            intArr.add(Byte.toUnsignedInt(b));
+        }
+        StringBuilder result = new StringBuilder();
+        result.append("[");
+        for (Integer integer : intArr) {
+            result.append(integer).append(",");
+        }
+        result.deleteCharAt(result.length() - 1);
+        result.append("]");
+        return result.toString().getBytes();
+    }
+
+    public static byte[] convertBytesToStringBytes(byte[] inputByte) {
+        return convertBytesToStringBytes(StringUtil.bytesToHex(inputByte));
+    }
```

Comment:
The conversion process seems to be redundant since it involves converting inputBytes into a hexformat initially, only to convert the resulting hex string back into bytes using convertBytesToStringBytes(String).

```suggestion
    public static byte[] convertBytesToStringBytes(String hexString) {
        var bytes = StringUtil.hexToBytes(hexString);
        return convertBytesToStringBytes(bytes);
    }

    public static byte[] convertBytesToStringBytes(byte[] bytes) {
        StringBuilder result = new StringBuilder();
        result.append("[");
        for (byte b : bytes) {
            var _unsignedValue=Byte.toUnsignedInt(b);
            result.append(_unsignedValue).append(",");
        }
        result.deleteCharAt(result.length() - 1);
        result.append("]");
        return result.toString().getBytes();
    }
```